flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCUpsertTableSink.java

```diff
@@ -69,7 +69,8 @@ private JDBCUpsertTableSink(
 
 	private JDBCUpsertOutputFormat newFormat() {
 		if (!isAppendOnly && (keyFields == null || keyFields.length == 0)) {
-			throw new UnsupportedOperationException("JDBCUpsertTableSink can not support ");
+			throw new UnsupportedOperationException("JDBCUpsertTableSink doesn't support append mode or upsert " +
```

Comment:
Hi @link3280 , `JDBCUpsertTableSink` can support append mode. It just not support upsert mode without keys.

Reply:
@JingsongLi Thanks for pointing that out. It's now fixed.